website/docs/reference/resource-properties/arguments.md

```diff
@@ -16,7 +16,7 @@ macros:
   - name: <macro name>
     arguments:
       - name: <arg name>
-        type: <string>
+        type: <column>
```

Comment:
this should still say <string> !

Reply:
we could even link to the supported types docs -> https://docs.getdbt.com/reference/resource-properties/arguments#supported-types